src/Node/index.tsx

```diff
@@ -49,13 +51,18 @@ export default class Node extends React.Component<NodeProps, NodeState> {
     initialStyle: {
       opacity: 0,
     },
+    wasClicked: false,
   };
 
   componentDidMount() {
     this.commitTransform();
   }
 
   componentDidUpdate() {
+    if (this.state.wasClicked) {
+      this.props.centerNode && this.props.centerNode(this.props.hierarchyPointNode);
```

Comment:
I think we can drop the initial `this.props.centerNode` truthy check since `props.centerNode` will always be defined right? The actual key condition `if (this.props.dimensions)...` lives inside the `centerNode` method itself in `Tree`.

```suggestion
      this.props.centerNode(this.props.hierarchyPointNode);
```